packages/discord.js/src/structures/VoiceChannelEffect.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const { Emoji } = require('./Emoji');
+
+/**
+ * Represents an effect used in a {@link VoiceChannel}.
+ */
+class VoiceChannelEffect {
```

Comment:
Shouldnt it extend `Base`
> like tetie said

Reply:
Probably not. Extending `Base` would mean this would (not publically) add `_clone()`, `_patch()`, and `_update()` to the class which are all irrelevant methods. In addition, `valueOf()` for this structure would return `undefined` as there is no `id` sent in voice channel effects.